README.md

```diff
@@ -1,3 +1,6 @@
+<p align="center"><img width=30% src="https://github.com/amrllkmn/healthyhome/blob/gh-pages/logo.png"></p>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
How come all of this extra whitespace?

Reply:
Again, the auto-formatting on my vscode.

Reply:
For all the `&nbsp;`? That's interesting, I wonder why is that!

Reply:
Oh yeah, that's what I was referring to! 😆 HTML whitespace tag!

Reply:
I think the logo looks good on my end. Always open to any changes and improvements.

Reply:
The file path for the logo is now ```docs/images/Logo_no_background.svg```. The ```docs``` as discussed can now be used to add other files that are not coding-related.